hypha/settings/base.py

```diff
@@ -323,6 +323,23 @@
 
 SHORT_DATETIME_FORMAT = 'Y-m-d\TH:i:s'
 
+DATETIME_INPUT_FORMATS = [
+    '%Y-%m-%d %H:%M:%S',     # '2006-10-25 14:30:59'
+    '%Y-%m-%d %H:%M:%S.%f',  # '2006-10-25 14:30:59.000200'
+    '%Y-%m-%d %H:%M',        # '2006-10-25 14:30'
+    '%Y-%m-%dT%H:%M',        # '2006-10-25T14:30 (this is extra)'
+    '%Y-%m-%d',              # '2006-10-25'
+    '%m/%d/%Y %H:%M:%S',     # '10/25/2006 14:30:59'
+    '%m/%d/%Y %H:%M:%S.%f',  # '10/25/2006 14:30:59.000200'
+    '%m/%d/%Y %H:%M',        # '10/25/2006 14:30'
+    '%m/%d/%Y',              # '10/25/2006'
+    '%m/%d/%y %H:%M:%S',     # '10/25/06 14:30:59'
+    '%m/%d/%y %H:%M:%S.%f',  # '10/25/06 14:30:59.000200'
+    '%m/%d/%y %H:%M',        # '10/25/06 14:30'
+    '%m/%d/%y',              # '10/25/06'
+]
```

Comment:
Confirming that Chrome gets a validation error without the '%Y-%m-%dT%H:%M' format. I thought it worked for me before but I must have been sloppy in my testing, you definitely was correct.

By adding the '%Y-%m-%dT%H:%M' here we do not need to add it to the form field. The forms then works in both Chrome and Safari.

Reply:
Looks good, thanks. It will help us to use default input formats along with extra with T.